D06 - Protect Secrets.md

```diff
@@ -2,8 +2,15 @@
 
 
 ## Threat Scenarios
+- in worst Case Secrets would be stored somewhere in our Container Environment
+- exposed environment variables 
+- priviliged access, in worst case as root user
 
 ## How Do I prevent?
+- running containers as a non-root user
+- using multi-stage builds
```

Comment:
This probably has also nothing to do with it unless you have in the first stage a credential which is futile, right?